internal/cmd/manager/instance/run/cmd.go

```diff
@@ -105,7 +107,8 @@ func NewCmd() *cobra.Command {
 		"current cluster in k8s, used to coordinate switchover and failover")
 	cmd.Flags().StringVar(&namespace, "namespace", os.Getenv("NAMESPACE"), "The namespace of "+
 		"the cluster and of the Pod in k8s")
-
+	cmd.Flags().BoolVar(&tlsStatus, "tls-status", false,
```

Comment:
IMO `tls-status` makes it seem like "status of the TLS".
Like my other comment, I'd suggest `status-port-tls` or `status-via-tls` or `enable-status-via-tls`

Reply:
I would use `--status-port-tls`